src/fides/api/ctl/migrations/versions/c9ee230fa6da_add_config_set_column_to_application_.py

```diff
@@ -22,10 +24,28 @@ def upgrade():
         sa.Column(
             "config_set",
             sqlalchemy_utils.types.encrypted.encrypted_type.StringEncryptedType(),
-            nullable=False,
+            nullable=True,  # keep it nullable to start, in case we have any existing records
```

Comment:
Alternatively, if you want to keep an existing applicationconfig record instead of deleting, you can provide a server default, and then get rid of that default later.  so here, something like:

```python
from sqlalchemy.dialects.postgresql import JSON
from fides.core.config import CONFIG

encryptor = sqlalchemy_utils.types.encrypted.encrypted_type.StringEncryptedType(
    JSONTypeOverride,
    CONFIG.security.app_encryption_key,
    AesGcmEngine,
    "pkcs5",
)
val = encryptor.process_bind_param({}, JSON)

op.add_column(
    "applicationconfig",
    sa.Column(
        "config_set",
        sqlalchemy_utils.types.encrypted.encrypted_type.StringEncryptedType(),
        nullable=False,
        server_default=val,
    ),
)

...and then at the end....
op.alter_column(
   "applicationconfig", "config_set", nullable=False, server_default=None
)

```

Reply:
oh wait, my whole reasoning for going the way i did (deletion) was because i'd assumed we don't have access to our fides `CONFIG` in the migration context. but maybe that assumption is wrong? i should have at least tried to verify it. i'll give it a try now - thanks for suggesting this!

Reply:
it looks like this does indeed work! thank you again for the pointer here @pattisdr . my new commit uses this approach - i don't even think we need to get rid of the default later, it will be overwritten naturally on server bootup, which is fine.

Reply:
By the default, I'm not talking about the row contents, I mean this will still be value postgres considers its server default. I'd personally revert the server default right after since this was specifically for migration purposes only. 

Benefits of reverting back to null means if somewhere in the application tries to set this to applicationconfig_set to nullin the future, we'll be prevented from doing that at the database level, instead of it being set to an encrypted empty dict.

Current state here:
```
                                               Table "public.applicationconfig"
   Column   |           Type           | Collation | Nullable |                            Default                            
------------+--------------------------+-----------+----------+---------------------------------------------------------------
 id         | character varying(255)   |           | not null | 
 created_at | timestamp with time zone |           |          | now()
 updated_at | timestamp with time zone |           |          | now()
 api_set    | character varying        |           | not null | 
 single_row | boolean                  |           | not null | 
 config_set | character varying        |           | not null | 'QrPxq7t+8fWmVwJ/47xkFnYEhNOEuQyIlVPLJOou'::character varying

```

Reply:
got it - i didn't totally grasp what a `server_default` was, so this makes sense -- thank you! i'll update accordingly.

Reply:
updated in latest commit - let me know if it looks good to you @pattisdr! did some manual validation locally too.